### PR TITLE
Automated Test - COMMERCE-6086 | master

### PR DESCRIPTION
**Pull request by leilanyaragao**

### CODE REVIEW

**Comment by leilanyaragao** [comment]
https://issues.liferay.com/browse/COMMERCE-6086, cc: @PietroBocale

**Comment by kristinonias** [comment]
ci:test:sf

**Comment by brittneyq** [comment]
Hi @leilanyaragao , can you remove the **@ignore = true** in the test stub for your test? Thank you!

**Comment by kristinonias** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductVisibility.testcase [hunk 1]
```diff
@@ -48,12 +48,71 @@ definition {
...
+		Click(locator1 = "ToggleSwitch#ENABLE_SPECIFIC_TOGGLE_SWITCH");
+        
+		Click(locator1 = "Button#PLUS");
+        
+		SelectFrame.selectFrameNoLoading(
+            locator1 = "IFrame#MODAL_BODY");
+        
+		Check(
+            locator1 = "Checkbox#ENABLE_CHECKBOX_SELECT_CHANNEL",
+            key_checkboxName = "Test Channel");
```
Comment (Pietro69) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductVisibility.testcase]: It's possible to use Checkbox#ENABLE_CHECKBOX with key_checkboxName = "Test Channel"

File: portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/Checkbox.path [hunk 2]
```diff
@@ -498,6 +498,11 @@
 	<td>//label[contains(.,'${key_checkboxOption}')]/input[@type='checkbox']/../span</td>
 	<td></td>
 </tr>
+<tr>
+<td>ENABLE_CHECKBOX_SELECT_CHANNEL</td>
+    <td>//tr[contains(@data-name,'${key_checkboxName}')]//input[contains(@type,'checkbox')]</td>
+    <td></td>
+</tr>
```
Comment (Pietro69) [inline portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/Checkbox.path:505]: I think is possible to remove it

File: portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/ToggleSwitch.path [hunk 3]
```diff
@@ -38,6 +38,11 @@
 	<td>//label[contains(@for,'enableSocialBookmarks')]/input/../span/span</td>
 	<td></td>
 </tr>
+<tr>
+    <td>ENABLE_SPECIFIC_TOGGLE_SWITCH</td>
+    <td>//span[contains(@id,'ojma___toggle-switch')]</td>
+    <td></td>
+</tr>
```
Comment (Pietro69) [inline portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/ToggleSwitch.path:45]: Please change this path and make it more reusable using variables

**Review by Pietro69 — CHANGES_REQUESTED** [review]

**Comment by Pietro69** [comment]
Please add the property `test.name.skip.portal.instance = "CPCommerceProductVisibility#AddAChannelFilter"`

**Comment by pedro-michel** [comment]
ci:test:sf

**Comment by Pietro69** [comment]
ci:test:commerce-functional

File: portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/ToggleSwitch.path [hunk 4]
```diff
@@ -38,6 +38,11 @@
 	<td>//label[contains(@for,'enableSocialBookmarks')]/input/../span/span</td>
 	<td></td>
 </tr>
+<tr>
+    <td>ENABLE_SPECIFIC_TOGGLE_SWITCH</td>
+    <td>//span[contains(@id,'${key_toggleSwitch}')]</td>
```
Comment (Pietro69) [inline portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/ToggleSwitch.path:43]: Please use this path `//h4[contains(.,"${key_name}")]//span[contains(@id,'toggle-switch')]`

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductVisibility.testcase [hunk 5]
```diff
@@ -48,12 +48,58 @@ definition {
...
+
+		CPCommerceCatalog.newProductsEntry(
+			productCatalog = "Master",
+			productName = "Simple T-Shirt",
+			productType = "Simple");
+
+		CommerceEntry.gotoMenuTab(menuTab = "Visibility");
+
+		Click(
+			key_toggleSwitch = "ojma___toggle-switch",
```
Comment (Pietro69) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductVisibility.testcase:80]: Please use key_name = "Channels"

**Review by Pietro69 — CHANGES_REQUESTED** [review]

**Comment by pedro-michel** [comment]
ci:test:sf

**Comment by Pietro69** [comment]
ci:test:commerce-functional

**Comment by Pietro69** [comment]
ci:test:commerce-functional

**Comment by kristinonias** [comment]
ci:test:commerce-functional

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductVisibility.testcase [hunk 6]
```diff
@@ -48,12 +48,57 @@ definition {
...
+			category = "Store Management",
+			portlet = "Channels");
+
+		Click(locator1 = "Button#PLUS");
+
+		CommerceEntry.addNewChannelsEntry(
+			channelCurrency = "US Dollar",
+			channelName = "Test Channel",
+			channelType = "Site",
+			siteName = "Guest");
```
Comment (kristinonias) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductVisibility.testcase:66]: The test is failing at this Macro. Please fix and resend.

**Review by kristinonias — CHANGES_REQUESTED** [review]

**Comment by p-albuquerque** [comment]
New PR sent #1222

**Pull request closed without merging**